src/hb-graphite2.cc

```diff
@@ -41,7 +41,11 @@
  * @short_description: Graphite2 integration
  * @include: hb-graphite2.h
  *
- * Functions for using HarfBuzz with the Graphite2 fonts.
+ * Functions for using HarfBuzz with fonts that include Graphite2 features.
```

Comment:
```suggestion
 * Functions for using HarfBuzz with fonts that include Graphite features.
```
AFAIK the `2` is for the new redesigned library, but Graphite fonts don’t have a major number change.